src/Spice86.Core/Emulator/ReverseEngineer/ArgumentFetcher.cs

```diff
@@ -0,0 +1,102 @@
+namespace Spice86.Core.Emulator.ReverseEngineer;
+
+using Spice86.Core.Emulator.CPU;
+using Spice86.Core.Emulator.Memory;
+using Spice86.Shared.Utils;
+
+/// <summary>
+/// Helper to get function arguments from the stack.
```

Comment:
The fact that this probably works best with the C calling convention (cdecl) should be mentionned.

Reply:
True, for other conventions the numbering in the argument names may be reversed.